app/views/feeds/show.atom.builder

```diff
@@ -1,3 +1,5 @@
+@feed.update(last_fetched: Time.zone.now)
```

Comment:
I have no idea if this is the right place for this—my feeling is that builders aren't supposed to have side effects?—but obv I am not a ruby/rails person.

Reply:
The convention is to put this in FeedsController#show, probably right after assigning `@feed`. Controllers can have side effects. 👍🏻